services/identity/SitesService.ts

```diff
@@ -29,6 +29,22 @@ class SitesService {
     return site
   }
 
+  async create(
+    createParams: Partial<Site> & {
+      name: Site["name"]
+      apiTokenName: Site["apiTokenName"]
+      creator: Site["creator"]
+    }
+  ) {
+    return this.repository.create(createParams)
+  }
```

Comment:
I think we might not need `apiTokenName` - it doesn't exist on Sites anymore

Reply:
Yes, this should be removed. I have added this as an item to #461.